classes/Visualizer/Module/Admin.php

```diff
@@ -1093,6 +1093,10 @@ public function renderLibraryPage() {
 
 		$this->load_survey();
 
+		if ( ! apply_filters( 'visualizer_is_business', false ) ) {
```

Comment:
Are we 100% sure that this is the only condition that marks the product as pro? To me it sounds that this only applies to the business plan. 

I am not that familiar with this plugin's codebase so maybe it's worth checking other places where free-only things are rendered.

For reference, this is the check we do when adding the pro upsell: 

https://github.com/Codeinwp/visualizer/blob/b1c71ac778ecacfd74263b82e4c6e006adb8cd01/classes/Visualizer/Module/Sources.php#L138-L139

Reply:
The code for that is here https://github.com/Codeinwp/visualizer-pro/blob/77e1836ad404589168902b90e4cfaba36e5927f6/inc/addon.php#L2255-L2274

If the filters return false, that the user is free and does not have the Pro module, or if it has the pro module, false if it  is below 1 or equal to 10.

If it returns true, then we have a pro user that is above Tier 1 Pro

Reply:
Thanks for the explanation! 👍🏻